tests/Client/Response/AddUserToGroupResponseTest.php

```diff
@@ -19,23 +19,19 @@
 /**
  * @author Serhiy Lunak <serhiy.lunak@gmail.com>
  */
-class AddUserToGroupResponseTest extends TestCase
+final class AddUserToGroupResponseTest extends TestCase
 {
     public function testCenBeCreated(): void
     {
-        $successfulCurlResponse = '{"status":1,"request":"aaaaaaaa-1111-bbbb-2222-cccccccccccc"}';
-        $response = new AddUserToGroupResponse($successfulCurlResponse);
-
+        $response = new AddUserToGroupResponse('{"status":1,"request":"aaaaaaaa-1111-bbbb-2222-cccccccccccc"}');
```

Comment:
I hate saying it, but I would actually like to preserve the variables there e.g. `$successfulCurlResponse`, `$unSuccessfulCurlResponse`. They serve purpose and provide context to understand what kind of response it is going to be. I am ok removing them only if we could rename `$response` to `$successfulRsponse` and so on. So either do not refactor that part at all or rename `$response`.

Reply:
Actually this is very good idea, thank you! Splitting tests with successful and unsuccessful cases into multiple tests?

Reply:
Exactly

Reply:
Can you please check again? Thanks